parlai/core/params.py

```diff
@@ -138,6 +138,19 @@ def add_parlai_args(self):
             '-bs', '--batchsize', default=1, type=int,
             help='batch size for minibatch training schemes')
         self.add_parlai_data_path(parlai)
+        # Find which task specified, and add its specific arguments.
```

Comment:
can you make a separate function, "add_task_args"  (like we have a "add_model_args" for this)